File: Beakjoon/9498.swift
// 시험 성적

import Foundation

let input = readLine()!
let inputToInt = Int(input)!

var result: String

if inputToInt >= 90 {
    result = "A"
}
else if inputToInt >= 80 {
    result = "B"
}
else if inputToInt >= 70 {
    result = "C"
}
else if inputToInt >= 60 {
    result = "D"
}
else {
    result = "F"
}

print(result)

File: Beakjoon/9498.py
# 시험 성적

import sys

input = sys.stdin.readline

inputData = int(input())

if inputData >= 90:
    answer = "A"
elif inputData >= 80:
    answer = "B"
elif inputData >= 70:
    answer = "C"
elif inputData >= 60:
    answer = "D"
else:
    answer = "F"

print(answer)